nrf_modem/doc/dectphy.rst

```diff
@@ -276,20 +280,19 @@ The application can schedule an RSSI measurement by operation by calling the :c:
 
 The operation is asynchronous, and the completion of the :c:func:`nrf_modem_dect_phy_rssi` function only signals that the request was sent to the modem.
 
-RSSI measurements are sent to the application in the :c:member:`nrf_modem_dect_phy_callbacks.rssi` callback function at a configurable interval,
-as specified by the :c:member:`nrf_modem_dect_phy_rssi_params.reporting_interval` field in the operation parameters.
+RSSI measurements are sent to the application in the :c:member:`nrf_modem_dect_phy_callbacks.rssi` callback function at a configurable interval, as specified by the :c:member:`nrf_modem_dect_phy_rssi_params.reporting_interval` field in the operation parameters.
 
-When the operation has completed, its result is signalled to the application in the :c:member:`nrf_modem_dect_phy_callbacks.op_complete` callback function.
+When the operation has completed, its result is signaled to the application in the :c:member:`nrf_modem_dect_phy_callbacks.op_complete` callback function.
 If any error has occurred in scheduling or executing the operation, it is returned in the callback.
 
-Combined TX/RX operations
-=========================
+TX and RX operations
```

Comment:
I put "Combined" in the title because it reflect the fact the two operations are combined (scheduled together).
"TX and RX operations" is a bit confusing because the previous sections in the page are "Transmission" and "Reception", that is, TX and RX, so it is not clear what is the difference between this section and those above. Would it be possible to keep "combined" in the title, why was it removed?

Reply:
could it be simply a "Combined TX/RX operation" not operation**s** as it's scheduled with single service

Reply:
or more clearly like "Combined **TX_RX** operation" to emphasize the execution order of the combined operations.